config/prod.exs

```diff
@@ -52,6 +52,9 @@ config :logger, level: :info
 #
 # Check `Plug.SSL` for all available options in `force_ssl`.
 
+config :jsonapi,
+  host: System.get_env("HOST_URL")
```

Comment:
How about moving this to prod.secrct file?

Reply:
Used to add this from Omise project 😛 Updated d3ddd77